scripts/test-tls13-mlkem.py

```diff
@@ -62,7 +63,7 @@ def help_msg():
 def main():
     host = "localhost"
     port = 4433
-    num_limit = None
+    num_limit = 400
```

Comment:
Later (line 421), there is a check `if not num_limit`. Do we want to keep that for cases when the test is invoked with `-n 0`?

Reply:
precisely that, yes